src/SystemLoader.cc

```diff
@@ -46,6 +46,15 @@ class gz::sim::SystemLoaderPrivate
               const sdf::ElementPtr &/*_sdf*/,
               gz::plugin::PluginPtr &_plugin)
   {
+    // Deprecated: accept ignition-gazebo-prefixed systems
+    auto filename = _filename;
+    if (filename.find("ignition-gazebo") != std::string::npos)
+    {
+      filename.replace(0, 15, "gz-sim");
+      gzwarn << "Trying to load deprecated plugin [" << _filename
+             << "]. Using [" << filename << "] instead." << std::endl;
+    }
+
```

Comment:
Is there a need to be accounting for `libignition-xxx` filenames as well?
It could break quite badly if `libignition-gazebo-xxx` -> `gz-simebo-xxx`

Reply:
Perhaps?

```suggestion
    // Deprecated: accept ignition-gazebo-prefixed systems
    auto filename = _filename;
    auto pos = filename.find("ignition-gazebo");
    if (pos != std::string::npos)
    {
      filename.replace(pos, pos+15, "gz-sim");
      gzwarn << "Trying to load deprecated plugin [" << _filename
             << "]. Using [" << filename << "] instead." << std::endl;
    }

```

Reply:
Good call, will do 👍🏽